### PR TITLE
DRF pagination rfop & ruo

### PR DESCRIPTION
**Pull request by Vital-jan**

### CODE REVIEW

File: ratu/views.py [hunk 1]
```diff
@@ -9,17 +9,50 @@
...
+            'data': data
+        }
+        return Response(data)
 
-class RuoView(APIView):
+class RuoView(GenericAPIView):
+    serializer_class = RuoSerializer
+    queryset = Ruo.objects.all()
+    pagination_class = CustomPagination
     def get(self, request):
```
Comment (Iva-khar) [inline ratu/views.py:43]: Why you define the same method in different classes?

Comment (criticus) [reply ratu/views.py:43]: +1 if you're reusing the same method it would make sense to move it to abstract class

File: ratu/views.py [hunk 2]
```diff
@@ -9,17 +9,50 @@
...
+        page = self.paginate_queryset(queryset)
+        if page is not None:
+            serializer = self.get_serializer(page, many=True)
+            result = self.get_paginated_response(serializer.data)
+            data = result.data # pagination data
+        else:
+            serializer = self.get_serializer(queryset, many=True)
+            data = serializer.data
+        payload = {
+            'return_code': '0000',
```
Comment (criticus) [inline ratu/views.py:33]: what does this code mean?

Comment (Vital-jan) [reply ratu/views.py:33]: API response pagination

File: ratu/views.py (hunk 1 above)
Comment (Vital-jan) [reply ratu/views.py:43]: ++

File: ratu/views.py (hunk 2 above)
Comment (Iva-khar) [reply ratu/views.py:30]: What means '0000'?

Comment (Vital-jan) [reply ratu/views.py:30]: It`s built-in django value.

**Pull request closed without merging**